docs/Validation Rules.md

```diff
@@ -1,17 +1,15 @@
-# Implemented validation rules
```

Comment:
@scottclowe What is the main reason to change from the markdown way of creating headings to use the rst way of creating headings? Also that you have mixed heading types inside the document, both md and rst O.o

I would prefer to stick with markdown as it works the best on github. Or do you have some other reason in mind?